server/config/local.json-dist

```diff
@@ -17,6 +17,7 @@
     "fmt": "pretty",
     "level": "debug"
   },
+  "scopedKeysEnabled": true,
```

Comment:
This needs to be called out for local devs!

Reply:
Sorry, what does this mean?

Reply:
it means we are all going to have to update our local.json for scoped keys to work.

Reply:
Ah yeap yeap 😄